redisinsight/ui/src/components/query-card/QueryCardHeader/QueryCardHeader.tsx

```diff
@@ -75,11 +79,14 @@ const QueryCardHeader = (props: Props) => {
   }
 
   const sendEvent = (event: TelemetryEvent, query: string, additionalData: object = {}) => {
+    const workbenchMode = localStorageService?.get(BrowserStorageItem.workbenchMode)
```

Comment:
why do you get mode from storage if you have mode as props variable?

Reply:
Because it is props for query card, but for sending command we use actual editor's "raw mode" state